integration/tests/iam_v1_to_v2_force_upgrade.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash
 
+# this test script:
+# 1. deploys an older version of Automate on IAM v1.
+# 2. runs inspec tests to verify IAM v1 behavior.
```

Comment:
What's the point of running the inspec tests on the old install? Are we using it as a health check?

Reply:
Yeah I was thinking they'd be helpful as a sanity check and a set of v1 behavior expectations we could refer to.